src/platform_impl/android/mod.rs

```diff
@@ -157,7 +157,7 @@ impl EventLoop {
 }
 
 impl EventLoopProxy {
-    pub fn wakeup(&self) -> Result<(), ::EventLoopClosed> {
+    pub fn wakeup(&self) -> Result<(), ::EventLoopClosed<()>> {
```

Comment:
I haven't found an appropriate `send_event` method for Android, this was the most-similiar method.